src/overlays/actors/ovl_En_Wiz_Brock/z_en_wiz_brock.h

```diff
@@ -8,10 +8,14 @@ struct EnWizBrock;
 typedef void (*EnWizBrockActionFunc)(struct EnWizBrock*, GlobalContext*);
 
 typedef struct EnWizBrock {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x5C];
+    /* 0x0000 */ DynaPolyActor dyna;
+    /* 0x015C */ char unk_15C[0x44];
```

Comment:
```suggestion
    /* 0x015C */ UNK_TYPE1 unk_15C[0x44];
```
Also seems like the leading 0 can be dropped from the first three offset comments.

Reply:
Disappointing there's no unused collider in here to fit this.

Reply:
I _think_ this is used in en_wiz, I'll change it to `UNK_TYPE` but I think it will be filled in.